src/DynamoCore/Search/SearchElements/ZeroTouchSearchElement.cs

```diff
@@ -16,16 +16,17 @@ namespace Dynamo.Search.SearchElements
     public class ZeroTouchSearchElement : NodeSearchElement
     {
         private readonly FunctionDescriptor functionDescriptor;
+        private readonly string fullname;
 
         /// <summary>
         /// The name that is used during node creation
         /// </summary>
         public override string CreationName { get { return functionDescriptor != null ? functionDescriptor.MangledName : this.Name; } }
 
         /// <summary>
-        ///     The full name of entry which consists of qualified name for function descriptor.
+        ///     The full name of entry which consists of assembly name and qualified name for function descriptor.
         /// </summary>
-        public override string FullName { get { return functionDescriptor != null ? functionDescriptor.QualifiedName : base.FullName; } }
+        public override string FullName { get { return fullname; } }
```

Comment:
As a side note, this will likely affect other callers of `FullName` (maybe in-canvas menu?), just want to make sure those are also not broken because of the new name.